api/blog/serializers.py

```diff
@@ -7,7 +7,7 @@
 from .models import Post
 
 
-class TagSerializer(serializers.HyperlinkedModelSerializer):
+class TagSerializer(serializers.ModelSerializer):
```

Comment:
Right now, the details for an individual tag are obtained at `tags/tag_id`. Since we don't have this info exposed anywhere, we would need to get the tag details at `tags/tag_slug`.

Reply:
so revert this ?

Reply:
Not sure if it's necessary. I just want the `lookup_field` to be `slug` the same way it is for the posts.

Reply:
done